Voog/design-berlin
# Add sale price support

Adds sale price support, which includes:
- Displaying sale prices on auto-rendered product pages
- Displaying sale prices on product list layout
- Making sale prices react to design editor changes
- Making sale prices react to dark backgrounds

Closes #129 

Testing:
- Make sure that sale prices are displayed on auto-rendered product pages
- Make sure that sale prices are present on Product List layout
- Prices react to dark backgrounds & design editor changes
